server/repository/userRepository.js

```diff
@@ -17,6 +17,38 @@ class UserRepository {
     return newUser;
   }
 
+  /**
+   * Updates a user password resetToken
+   * @param {string} email
+   * @param {string} resetToken
+   * @returns {object} user
+   */
+  static async updateUserByEmail(email, resetToken) {
```

Comment:
updateUserByEmail isn't a very good name for this function.

Reply:
I will come up with a new name